tutorials/katacoda/1-serving-intro/step2.md

```diff
@@ -0,0 +1,26 @@
+## Deploy and autoscale application
+
+We are going to deploy the [Hello world sample web application](https://knative.dev/docs/serving/samples/hello-world/helloworld-go/). This basic web application reads in an env variable TARGET and prints `Hello ${TARGET}!`. If TARGET is not specified, it will use `World` as the TARGET.
+
+We will now deploy the application by specifying the image location and the `TARGET` env variable.
+
+Knative defines a `service.serving.knative.dev` CRD to control the lifecycle of the application (not to be confused with kubernetes service). We will use the `kn` cli to create the Knative service:
+
+```
+kn service create demo --image gcr.io/knative-samples/helloworld-go --env TARGET=blue
+```{{execute}}
+
+We can now invoke the application using `curl`. We first need to figure out the IP address of minikube and the ingress port.
+```
+MINIKUBE_IP=$(minikube ip)
+INGRESS_PORT=$(kubectl get svc envoy --namespace contour-external --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')
+```{{execute}}
+Then invoke the application using curl:
+```
+curl http://$MINIKUBE_IP:$INGRESS_PORT/ -H 'Host: demo.default.example.com'
```

Comment:
It would be great to configure the minikube to use a local domain name such as `http://demo.default.127.0.0.1.nip.io`
Or a local IP `10.` when I use minikube I use `minikube tunnel` to get a local tunnel on an IP address for a load balancer IP.

Reply:
I think we can fix that in a follow on PR.